system/content/models/Content.js

```diff
@@ -0,0 +1,24 @@
+var mongoose = require('mongoose');
+
+//TODO complete this schema!
+// review if is better to have an Embedded document pattern where
+// {store: id, content:[{item}, {item}, {item}]}
+var contentSchema = mongoose.Schema({
+  name: {
+    type: String,
+    required: [true, 'How do they order this content?']
+  },
+  _store: {
+    type: String,
```

Comment:
Should be of type [`ObjectId`](http://mongoosejs.com/docs/api.html#schema-objectid-js)